models/intel/person-detection-action-recognition-0006/description/person-detection-action-recognition-0006.md

```diff
@@ -21,37 +21,38 @@ This is an action detector for the Smart Classroom scenario. It is based on the
 | Min pedestrian height             | 80 pixels (on 1080p)                      |
 | GFlops                            | 8.225                                     |
 | MParams                           | 2.001                                     |
-| Source framework                  | TensorFlow*                               |
+| Source framework                  | TensorFlow\*                              |
 
 Average Precision (AP) is defined as an area under the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
 curve.
 
 ## Inputs
 
-Name: `input`, shape: [1x400x680x3] - An input image in the format [BxHxWxC],
-   where:
-    - B - batch size
-    - H - image height
-    - W - image width
-    - C - number of channels
+Name: `input`, shape: `1, 400, 680, 3` - An input image in the format `B, H, W, C` where:
```

Comment:
hmm... looks like description of tf input format, not openvino
```suggestion
Name: `input`, shape: `1, 3, 400, 680` - An input image in the format `B, C, H, W` where:
```

Reply:
Updated